package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "ava",
-    "lint": "eslint test index.js"
+    "lint": "eslint test src/**/*.ts",
+    "build": "tsc -p ./"
```

Comment:
What was breaking that we needed to use `-p ./`?

Reply:
This should be reverted, as it's troubleshooting cruft.  

61a1b3e